### PR TITLE
docs: QUIC GA

### PR DESCRIPTION
**Pull request by alyssawilk**
Risk Level: n/a
Testing: n/a
Docs Changes: yes
Release Notes: yes

### CODE REVIEW

**Comment by moderation** [comment]
Woohoo :+1:

File: tools/docs/rst_check.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 INVALID_REFLINK = r".* ref:.*"
 REF_WITH_PUNCTUATION_REGEX = r".*\. <[^<]*>`\s*"
-VERSION_HISTORY_NEW_LINE_REGEX = r"\* ([a-z \-_]+): ([a-z:`]+)"
+VERSION_HISTORY_NEW_LINE_REGEX = r"\* ([0-9a-z \-_]+): ([a-z:`]+)"
```
Comment (RyanTheOptimist) [inline tools/docs/rst_check.py:11]: LOL

**Review by RyanTheOptimist — DISMISSED** [review]
Woo hoo!

**Comment by alyssawilk** [comment]
Thanks for triaging the upstream list! I've tried to update the docs to include where upstream is good to go - lmk what you think?

**Review by RyanTheOptimist — DISMISSED** [review]
Looks great. Thanks!

File: docs/root/intro/arch_overview/http/http3.rst [hunk 2]
```diff
@@ -5,16 +5,13 @@ HTTP3 overview
...
-  `here <https://github.com/envoyproxy/envoy/labels/quic-mvp>`_
-  For example QUIC does not currently support in-place filter chain updates, so users
-  requiring dynamic config reload for QUIC should wait until
-  `#13115 <https://github.com/envoyproxy/envoy/issues/13115>`_ has been addressed.
-
-  For general feature requests beyond production readiness, you can track
+  HTTP/3 downstream support is now GA, but continued improvements are coming, tracked in
   the `area-quic <https://github.com/envoyproxy/envoy/labels/area%2Fquic>`_ tag.
 
+  HTTP/3 upstream support is fine for in-datacenter use, but is not ready for internet use
```
Comment (danzh2010) [inline docs/root/intro/arch_overview/http/http3.rst:11]: in-datacenter or intra-?

Comment (alyssawilk) [reply docs/root/intro/arch_overview/http/http3.rst:11]: I really mean "in your datacenters or on your network" and I don't know how to phrase that :-/  Ideas?

Comment (RyanTheOptimist) [reply docs/root/intro/arch_overview/http/http3.rst:11]: How about something like "is fine for locally controlled networks, but is not ready for general internet use"?

**Review by RyanTheOptimist — APPROVED** [review]

File: docs/root/intro/arch_overview/http/http3.rst [hunk 3]
```diff
@@ -5,16 +5,13 @@ HTTP3 overview
...
 
-  HTTP/3 support is still in Alpha, and should be used with caution.
-  Outstanding issues required for HTTP/3 to go GA can be found
-  `here <https://github.com/envoyproxy/envoy/labels/quic-mvp>`_
-  For example QUIC does not currently support in-place filter chain updates, so users
-  requiring dynamic config reload for QUIC should wait until
-  `#13115 <https://github.com/envoyproxy/envoy/issues/13115>`_ has been addressed.
-
-  For general feature requests beyond production readiness, you can track
+  HTTP/3 downstream support is now GA, but continued improvements are coming, tracked in
```
Comment (ggreenway) [inline docs/root/intro/arch_overview/http/http3.rst:8]: Probably worth spelling out what GA stands for.

Comment (ggreenway) [reply docs/root/intro/arch_overview/http/http3.rst:8]: Also, should we note that hot-restart does not work with QUIC?

File: docs/root/intro/arch_overview/http/http3.rst (hunk 2 above)
Comment (ggreenway) [reply docs/root/intro/arch_overview/http/http3.rst]: Should we use the terminology used in other places for the docs, similar to `The following extensions are intended to be hardened against untrusted downstreams but assume trusted upstreams`?

Comment (alyssawilk) [reply docs/root/intro/arch_overview/http/http3.rst]: This has nothing to do with trusted or untrusted, it has to do with if your routing hardware will do funky things to UDP packets, like allow N on a given tuple and then black hole.

**Comment by alyssawilk** [comment]
swapping @lizan in for API shephard - I would like to get this one before the release.

**Review by ggreenway — APPROVED** [review]
LGTM! Exciting!

**Comment by danzh2010** [comment]
Can this be held for https://github.com/envoyproxy/envoy/pull/19495 and https://github.com/envoyproxy/envoy/pull/19494?

**Comment by alyssawilk** [comment]
Sure, ping when you're ready!
/wait

**Comment by danzh2010** [comment]
> Sure, ping when you're ready! /wait

#19495 and #19494 are merged. Thanks for waiting!

**Review by lizan — APPROVED** [review]
🎉

**Pull request merged**